app/controllers/users_controller.rb

```diff
@@ -0,0 +1,40 @@
+class UsersController < ApplicationController
+  def new
+    @user = User.new
+  end
+
+  def index
+    @users = User.all
+  end
```

Comment:
You don't need the index action in this exercise.

Reply:
We wanted to add an extra feature. We want to keep it if it is possible.